src/Status/StatusCountTable/StatusCountTable.types.ts

```diff
@@ -0,0 +1,16 @@
+import { SxProps, Theme } from "@mui/material";
+
+export type StatusCountTableProps = {
+  /**
+   * Title of the table
+   */
+  title: string;
+  /**
+   * Count is an object which contains objects with key status name and status count
+   */
+  count: { [status: string]: number };
```

Comment:
```suggestion
  count: { [status: Status]: number };
```

Reply:
I have received errors from typescript: An index signature parameter type cannot be a literal type or generic type. Consider using a mapped object type instead. I tried as well { [status in Status]: number } but then there is an error that status is not used.

Reply:
Interesting, `Record<Status, number>` should work instead.

Reply:
we can implement also as {[status in Status]: number }, but eslint throws an error that status in defined but never used